src/Umbraco.Web/PublishedContentExtensions.cs

```diff
@@ -1797,99 +1801,143 @@ public static IPublishedContent FirstChild(this IPublishedContent content)
         /// <returns>The first child of content, of the given content type.</returns>
         public static IPublishedContent FirstChild(this IPublishedContent content, string alias)
         {
-            return content.Children( alias ).FirstOrDefault();
+            return content.Children(alias).FirstOrDefault();
         }
 
+        /// <summary>
+        /// Gets the first child of the content using the passed predicate
+        /// </summary>
+        /// <param name="content">The content.</param>
+        /// <param name="predicate">The predicate query.</param>
+        /// <returns>The first child of content that matches the predicate.</returns>
         public static IPublishedContent FirstChild(this IPublishedContent content, Func<IPublishedContent, bool> predicate)
         {
             return content.Children(predicate).FirstOrDefault();
         }
 
+        /// <summary>
+        /// Gets the first child of the content, of a given content type, and returns it as <see cref="IPublishedContent"/>.
+        /// For strongly typed version use <seealso cref="FirstChildOf{T}(IPublishedContent)"/>.
+        /// </summary>
+        /// <param name="content">The content.</param>
+        /// <param name="alias">The content type alias.</param>
+        /// <returns>The first child of content, of the given content type.</returns>
         public static IPublishedContent FirstChild<T>(this IPublishedContent content)
             where T : class, IPublishedContent
         {
             return content.Children<T>().FirstOrDefault();
         }
 
+        /// <summary>
+        /// Gets the first child of the content using the passed predicate and returns it as <see cref="IPublishedContent"/>.
+        /// For strongly typed version use <see cref="FirstChildOf{T}(IPublishedContent, Func{T, bool})"/>.
+        /// </summary>
+        /// <param name="content">The content.</param>
+        /// <param name="predicate">The predicate query.</param>
+        /// <returns>The first child of content that matches the predicate.</returns>
         public static IPublishedContent FirstChild<T>(this IPublishedContent content, Func<IPublishedContent, bool> predicate)
             where T : class, IPublishedContent
         {
             return content.Children<T>().FirstOrDefault(predicate);
         }
 
+        /// <summary>
+        /// Gets the first child of the content, of a given content type and returns it as a strongly-typed model
+        /// </summary>
+        /// <param name="content">The content.</param>
+        /// <param name="alias">The content type alias.</param>
+        /// <returns>The first child of content, of the given content type as type T.</returns>
+        public static T FirstChildOf<T>(this IPublishedContent content)

```

Comment:
Think you meant `FirstChildAs<T>` here? ;-)

Reply:
Renamed in last commit :)